src/estimagic/optimization/subsolvers/_conjugate_gradient_quadratic.py

```diff
@@ -87,22 +96,22 @@ def _get_distance_to_trustregion_boundary(candidate, direction, radius):
 
     The positive distance sigma is defined in Eculidean norm, as follows:
 
-        `|| c + sigma * d || = radius`
+        || x + sigma * d || = radius
 
-    where `c` denotes the candidate vector, and `d` the direction vector.
+    where x denotes the candidate vector, and d the direction vector.
 
     Args:
         candidate(np.ndarray): Candidate vector of shape (n,).
         direction (np.ndarray): Direction vector of shape (n,).
         radius (floar): Radius of the trust-region
 
     Returns:
-        (float) Distance of the candidate vector to the trustregion
+        float: The candidate vector's distance to the trustregion
             boundary.
     """
-    cc = np.dot(candidate, candidate)
-    cd = np.dot(candidate, direction)
-    dd = np.dot(direction, direction)
+    cc = candidate.T @ candidate
```

Comment:
Do you really need the transposes? I guess candidate is a 1d array so `.T` has no effect?